index.html

```diff
@@ -8,31 +8,50 @@
     <link href="/fonts/css/all.css" rel="stylesheet" />
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
-      href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Varela+Round&display=swap"
       rel="stylesheet"
     />
   </head>
   <body class="app-grid">
     <header class="app-grid__header app-title">
-      <h1 class="app-title__headline">Quiz App</h1>
+      <h1 class="app-title__headline">Quizzy</h1>
     </header>
-    <main class="app-grid__main"></main>
+    <main class="app-grid__main">
+      <section class="questions">
+        <i class="fas fa-bookmark questions__bookmark"></i>
+        <h3 class="questions__title">What is the population of New Orleans?</h3>
+        <section class="questions__answer">
+          <details>
+            <summary><h3>Answer</h3></summary>
+            According to 2010 Census Bureau estimates, New Orleans' population
+            is made up of approximately 343,829 residents.
+          </details>
+          <ul class="questions__tags">
+            <li>Geography</li>
+            <li>Cities</li>
+            <li>USA</li>
+            <li>General</li>
+            <li>Globalisierung</li>
+          </ul>
+        </section>
+      </section>
+    </main>
     <footer class="app-grid__footer">
       <nav class="navigation">
-        <a href="">
-          <i class="fas fa-home"></i><br />
+        <a href="" alt="Home">
```

Comment:
The `alt`-attribute is for `img`-elements, not for anchors.
```suggestion
        <a href="" >
```